lib/css_parser/rule_set.rb

```diff
@@ -667,7 +667,7 @@ def parse_declarations!(block) # :nodoc:
         value = decs[(colon + 1)..]
         property.strip!
         value.strip!
-        next if property.empty? || value.empty?
+        next if property.empty? || value.empty? || value.start_with?(CssParser::IMPORTANT_IN_PROPERTY_RX)
```

Comment:
`value == CssParser::IMPORTANT` (with IMPORTANT being "!important")
would make it easier to read

Reply:
Done, except used `casecmp?` because the string, as most things in CSS, is case-insensitive.